test/unit/registry.test.ts

```diff
@@ -0,0 +1,52 @@
+import { expect } from 'chai';
+
+import { GithubRegistry } from '../../src/registry/GithubRegistry.js';
+import { LocalRegistry } from '../../src/registry/LocalRegistry.js';
+
+describe('Registry utilities', () => {
```

Comment:
Not really new, this is a modified `GithubRegistry.test.ts` but I guess git didn't make the connection